build.gradle

```diff
@@ -54,7 +54,7 @@ dependencies {
 	//	 ie³ power system utils
 	compile 'com.github.ie3-institute:PowerSystemUtils:1.5+'
 
-	implementation 'tech.units:indriya:2.0.4'
+	implementation 'tech.units:indriya:2.1.2'
```

Comment:
Why is this change needed? I'm a bit anxious about updating the units API, as it has brought a lot of bugs and side effects in the past.

Reply:
Indriya in versin 2.0.4 does not serialize some quantities properly. There's an error thrown because a transient field is not re-initialized. I picked 2.1.2 because it was the newest version and it fixed the issue, but if you like I can try to find an earlier version that works as well.